src/i18n/dictionaries/index.ts

```diff
@@ -5,3 +5,8 @@ export const dictionaries = {
 	default: english,
 	"pt-br": ptbr,
 };
+
+export const languages: { [key: string]: string } = {
+	en: "English",
+	"pt-br": "Português do Brasil",
```

Comment:
My only nib is that "Português do Brasil" will never fit in the dropdown
![Screenshot 2024-03-15 at 20 17 10](https://github.com/solidjs/solid-docs-next/assets/2382552/d9bf0d38-fa2a-4065-ad44-7d02358b1b79)

So maybe let's switch the language-naming pattern here.
I'd propose 2 alternatives

- Locale code `en` and `pt-br`
- Language when it's not localized (e.g.: English) and locale when it's localized (PT-BR), perhaps even with emoji flag (🇧🇷 PT-BR)

Wdyt?

Reply:
As discussed offline, added locale code when in English and fixed line break issue